benchmarks/benchmark_transform.py

```diff
@@ -1,14 +1,74 @@
+# See "Writing benchmarks" in the asv docs for more information.
+# https://asv.readthedocs.io/en/latest/writing_benchmarks.html
 import numpy as np
-from skimage import transform
+from scipy import ndimage as ndi
+from scipy.ndimage.morphology import distance_transform_edt
+from skimage.transform import generalized_distance_transform
 
-class TransformSuite:
-    """Benchmark for transform routines in scikit-image."""
 
+class edt2d:
+    """Benchmark for distance transform in scikit-image."""
+    timeout = 120.0
     def setup(self):
-        self.image = np.zeros((2000, 2000))
-        idx = np.arange(500, 1500)
-        self.image[idx[::-1], idx] = 255
-        self.image[idx, idx] = 255
+        self.case = (1+-1*(np.random.randint(50, size=(4096,4096))//48)).astype('float64')
 
-    def time_hough_line(self):
-        result1, result2, result3 = transform.hough_line(self.image)
```

Comment:
I think it is best if you create a new file with your benchmarks in it. You shouldn't remove this one.